binaries/speed_benchmark_torch.cc

```diff
@@ -200,7 +204,10 @@ int main(int argc, char** argv) {
     std::cout << module.forward(inputs) << std::endl;
   }
 
-  c10::WithCPUCachingAllocatorGuard cachine_allocator_guard;
+  std::unique_ptr<c10::CPUCachingAllocator> caching_allocator =
+    std::make_unique<c10::CPUCachingAllocator>();
```

Comment:
Maybe 

```
c10::CPUCachingAllocator caching_allocator;
```

instead of putting it on the heap?

Reply:
I thought it was cleaner to provide a pointer to caching_allocator from heap then from stack.